src/layouts/RabitLayout/Sidebar/index.tsx

```diff
@@ -20,11 +20,31 @@ const SidebarWrapper = styled(Box)(
         background  : ${theme.sidebar.background};
         box-shadow  : ${theme.sidebar.boxShadow};
         height      : 100%;
-                
+
+        @media (min-width: ${theme.breakpoints.values.xs}px) {
+            position                    : fixed;
+            z-index                     : 10;
+        }
+
+        @media (min-width: ${theme.breakpoints.values.sm}px) {
+            position                    : fixed;
+            z-index                     : 10;
+        }
+
+        @media (min-width: ${theme.breakpoints.values.md}px) {
+            position                    : fixed;
+            z-index                     : 10;
+        }
+
         @media (min-width: ${theme.breakpoints.values.lg}px) {
             position                    : fixed;
             z-index                     : 10;
         }
+
+        @media (min-width: ${theme.breakpoints.values.xl}px) {
+            position                    : fixed;
+            z-index                     : 10;
+        }
```

Comment:
Not sure if this is where the change is written, but the sidebar does not get smaller/hide when you reduce the window size. Only the RABIT logo and user box change:

**Example:**
![image](https://user-images.githubusercontent.com/101340211/192700558-1fa3850b-a1fe-4d0c-ab09-348678a0aec9.png)